src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -13,77 +13,105 @@
-    <% } %>
-    <a href="/about.jsp">About</a>
-  </nav>
+	<nav>
+		<a id="navTitle" href="/">CodeU Chat App</a> <a href="/conversations">Conversations</a>
+		<%
+		  if (request.getSession().getAttribute("user") != null) {
+		%>
+		<a>Hello <%=request.getSession().getAttribute("user")%>!
+		</a>
+		<%
+		  if ((boolean) request.getSession().getAttribute("isAdmin") == true) {
+		%>
+		<a href="/admin">Admin</a>
+		<%
+		  }
+		  } else {
+		%>
+		<a href="/login">Login</a>
+		<%
+		  }
+		%>
+		<a href="/about.jsp">About</a>
+	</nav>
 
-  <div id="container">
+	<div id="container">
 
-    <% if(request.getAttribute("error") != null){ %>
-        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
-    <% } %>
+		<%
+		  if (request.getAttribute("error") != null) {
+		%>
+		<h2 style="color: red"><%=request.getAttribute("error")%></h2>
+		<%
+		  }
+		%>
 
-    <% if(request.getSession().getAttribute("user") != null){ %>
-      <h1>New Conversation</h1>
-      <form action="/conversations" method="POST">
-          <div class="form-group">
-            <label class="form-control-label">Title:</label>
-          <input type="text" name="conversationTitle">
-        </div>
+		<%
+		  if (request.getSession().getAttribute("user") != null) {
+		%>
+		<h1>New Conversation</h1>
+		<form action="/conversations" method="POST">
+			<div class="form-group">
+				<label class="form-control-label">Title:</label> <input type="text"
+					name="conversationTitle">
+			</div>
 
-        <button type="submit">Create</button>
-      </form>
+			<button type="submit">Create</button>
+		</form>
 
-      <hr/>
-    <% } %>
+		<hr />
+		<%
+		  }
+		%>
 
-    <h1>Conversations</h1>
+		<h1>Conversations</h1>
 
-    <%
-    List<Conversation> conversations =
-      (List<Conversation>) request.getAttribute("conversations");
-    if(conversations == null || conversations.isEmpty()){
-    %>
-      <p>Create a conversation to get started.</p>
-    <%
-    }
-    else{
-    %>
-      <ul class="mdl-list">
-    <%
-      for(Conversation conversation : conversations){
-    %>
-      <li><a href="/chat/<%= conversation.getTitle() %>">
-        <%= conversation.getTitle() %></a></li>
-    <%
-      }
-    %>
-      </ul>
-    <%
-    }
-    %>
-    <hr/>
-  </div>
+		<%
+		  List<Conversation> conversations = (List<Conversation>) request.getAttribute("conversations");
+		  if (conversations == null || conversations.isEmpty()) {
+		%>
+		<p>Create a conversation to get started.</p>
+		<%
+		  } else {
+		%>
+		<ul class="mdl-list">
+			<%
+			  for (Conversation conversation : conversations) {
+			%>
+			<li><a href="/chat/<%=conversation.getTitle()%>"> <%=conversation.getTitle()%></a>
+				       <form method='POST'>
+					       <input align="right" onclick=change() type="button" value=Following name ="Following" id ="following"/> 
+					    </form>
+			<%
+			  }
+			%>
+		</ul>
+		<%
+		  }
+		%>
+		<hr />
+	</div>
 </body>
+<script>
+	function change()
```

Comment:
What does this function do?

Reply:
Switches the button from Following to Follow after it's clicked

Reply:
Ah, I think that's triggered before the function is submitted. Maybe try removing this function?